### DISCUSSION TITLE
複雜度

### DISCUSSION THREAD

**Discussion by jerry850313** [1101-GS4719-C/Python-Community-GS4719-C#58] [Ideas]
複雜度包含
  1.時間複雜度 :
     code執行所需時間
  2.空間複雜度 : 
     code執行所需記憶空間
=>一個好的程式勢必要讓這兩者越小越好 💯 
![image](https://jason-chen-1992.weebly.com/uploads/1/0/8/5/108557741/complexity_orig.png)
[參考資料](https://jason-chen-1992.weebly.com/home/time-space-complexity)

**Comment by enoch-lightning** [comment]
1.就我目前的理解，程式執行的時間，稱為「執行時間」即可。我的疑問：「時間複雜度」與「執行時間」有什麼不同的含意嗎？

**Reply by jerry850313** [reply]
時間複雜度可以單指某一段演算法的運算時間，執行時間是一整個code所執行的時間
